### DISCUSSION TITLE
"Real Time" Conversations in Github PR reviews

### DISCUSSION THREAD

**Discussion by mashton** [community/community#73148] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

My team uses comments and responses both inline, and in the main "thread" of a PR review extensively. We consider these conversations to be important context for future engineers, so we take steps to ensure robust and frequent conversation. Because of this, it's become painfully common for a conversation spanning hours or days or more to get quite disjointed. If I don't take care to refresh the page before I respond to a comment, I'm very likely to see a handful of comments show up after I submit. These were comments that were made between page refreshes, which if not done manually, do not seem to occur at all. 

If Github intends to support this type of workflow, would Github consider investing in improvements to it? My vague suggestion here would be to increase the "real time"-ness of PR conversations. At the very least, this could be something very similar to the "outdated" label comments get when new code as been pushed. At it's richest, it could be some sort of real-time conversation.
 
Thoughts?

**Comment by ericxu-echelon** [comment]
Bumping. Pushes are already real time, so it only makes sense to expect comments to show up in real time as well. I’ve often replied to a thread, only to find out someone had already done so between refreshes.
